mort_cal.py

```diff
@@ -0,0 +1,38 @@
+#fixed variables
+## Fixed variables are mostly unliked, as we don't have good reasons typically to make these
+## fixed vars. Usually, you want this passed in by a user or you want the application to read
+## the variables from a config file, where they can be modified instead
```

Comment:
I included some comments, but generally, comments are bad. They age and people will come back not knowing what a comment originally meant. 

> A comment is a failure to express yourself in code.  If you fail, then write a comment; but try not to fail.
- Robert Martin